packages/components/data-table/src/data-table.tsx

```diff
@@ -273,6 +280,11 @@ const DataTable = <Row extends TRow = TRow>(props: TDataTableProps<Row>) => {
         (tableRef.current.offsetWidth - tableRef.current.clientWidth)
       : undefined;
 
+  const updatedProps = {
```

Comment:
We can also do this inline when using the `DataRow` and we don't need an intermediate constant:
```tsx
<DataRow<Row>
  {...props}
  columns={columnsData}
  row={row}
  key={row.id}
  rowIndex={rowIndex}
```

Reply:
I missed this initially. 
Updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/95ca76b86725e211f2fc69748866c26db1019ca6 
Thank you.